packages/cosmic-swingset/test/test-make.js

```diff
@@ -43,3 +68,92 @@ test('make and exec', async t => {
     }),
   );
 });
+
+const query = async args => {
+  const parts = [];
+  const run = new Promise((resolve, reject) => {
```

Comment:
this isn't really an async function.
```suggestion
const query = args => {
  const parts = [];
  return new Promise((resolve, reject) => {
```